packages/server/graphql/public/types/TeamPromptMeeting.ts

```diff
@@ -19,6 +21,48 @@ const TeamPromptMeeting: TeamPromptMeetingResolvers = {
       return series
     }
     return null
+  },
+  prevMeeting: async ({meetingSeriesId, createdAt}, _args, {dataLoader}) => {
+    if (meetingSeriesId) {
```

Comment:
this isn't even a +1 but just a suggestion. I'm personally a big fan of flattening code hierarchy as it's easier to read. 

Doing this
```
if (!meetingSeriesId) return null

// everything else goes here
```

makes the code even easier to read. If you would like to dig more, here's a good overview of the early return pattern: https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
Ah yea, nice catch. I usually prefer this, too